src/amo/containers/App.js

```diff
@@ -73,15 +75,20 @@ export class AppBase extends React.Component {
       InfoDialogComponent,
       MastHeadComponent,
       children,
+      clientApp,
       i18n,
+      lang,
       location,
     } = this.props;
+    const isHomePage = location.pathname === `/${lang}/${clientApp}/`;
```

Comment:
Oh, I see. I'm so used to us forcing trailing slash so I didn't know that was valid. Will fix, thanks.